packages/components/MenuButton/FluentUIReactNativeMenuButton.podspec

```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'FluentUI-React-Native-MenuButton'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+
+  s.source           = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
+  s.swift_version    = "5"
```

Comment:
Maybe just be explicit that we want "5.0" here so minor version updates are more obvious/purposeful?